webclient/arch_z/templates/arch_z/dj/partials/dj_show_on_map.html

```diff
@@ -45,6 +45,6 @@
     $(id).prop('disabled', false);
     document.getElementById("pian_text").style.display = "none";
     document.getElementById("pian_select").style.display = "block";
-    $(id).prop('disabled', true);
+    //$(id).prop('disabled', true);
```

Comment:
@pesikj

Reply:
@jnihnat Ten řádek tam musí být, jinak je možné editovat Pian ručně, což dle požadavku @motyc nesmí být možné. Pokud potřebuješ Pian změnit, tak si prosím nejprve to pole aktivuj, potom změň obsah a pak ho opět deaktivuj. Tím pádem obsah pomocí mapy změníš, ale uživatel bude mít pole pro editaci zablokované.

Reply:
@pesikj Já jsem to teď zkoušel a problém je, že to nyní nefunguje ani při zvolení funkce "Připojit PIAN podle ID". Problém v #1174 byl v tom, že se pole samovolně aktivovalo při výběru PIAN z mapy.